doc/reference/hv-make-options.rst

```diff
@@ -0,0 +1,199 @@
+.. _hypervisor-make-options:
+
+Hypervisor Makefile Options
+###########################
+
+The ACRN hypervisor source code provides a makefile to build the ACRN
+hypervisor binary and associated components.
+
+Assuming that you are at the top level of the ``acrn-hypervisor`` directory,
+you can run the ``make`` command to start the build. See :ref:`acrn_configuration_tool` for information about required input files.
```

Comment:
Suggest more consistent line wrapping for future new topics.

Reply:
I wasn't picking on our "80 character wrap" recommendation, but you're right.